client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.scss

```diff
@@ -20,11 +22,67 @@
 	.pattern-large-preview__pattern-header {
 		margin-block-end: var(--pattern-large-preview-block-gap);
 	}
+
 	.pattern-large-preview__pattern-footer {
 		margin-block-start: var(--pattern-large-preview-block-gap);
 	}
 }
 
+.pattern-large-preview__pattern {
+	$pattern-large-preview-outer-border-radius: calc(var(--device-switcher-border-radius) / 2);
```

Comment:
The border radius doesn't seem quite right for mobile view 🤔

![Screenshot 2023-03-08 at 9 12 34 AM](https://user-images.githubusercontent.com/797888/223594355-5dac3f6c-96f2-422b-b873-48ab6d870b60.png)